scene/gui/tree.cpp

```diff
@@ -120,6 +120,10 @@ void TreeItem::_change_tree(Tree *p_tree) {
 		}
 
 		if (tree->selected_item == this) {
+			for (int i = 0; i < tree->selected_item->cells.size(); i++) {
+				tree->selected_item->cells.write[i].selected = false;
+			}
+
```

Comment:
You have committed this unrelated change in this PR. I don't think it was intentional?

Reply:
you can see from the commit history that I haven't learned how to use rebase yet)